testsuite/MDAnalysisTests/topology/test_pdb.py

```diff
@@ -72,7 +74,7 @@ class PDBBase(ParserBase):
     expected_attrs = ['ids', 'names', 'record_types', 'resids',
                       'resnames', 'altLocs', 'icodes', 'occupancies',
                       'bonds', 'tempfactors', 'chainIDs']
-    guessed_attrs = ['types', 'masses']
+    guessed_attrs = ['types', 'masses','elements']
```

Comment:
You may have to remove `elements` from `guessed_attrs` if they are not all present in the files for the tests that subclass `PDBBase`.